src/utilities/getIntrospectionQuery.ts

```diff
@@ -70,6 +80,19 @@ export function getIntrospectionQuery(options?: IntrospectionOptions): string {
     return optionsWithDefault.inputValueDeprecation ? str : '';
   }
   const oneOf = optionsWithDefault.oneOf ? 'isOneOf' : '';
+  function ofType(level: number, indent: string): string {
+    if (level <= 0) {
+      return '';
+    }
+    if (level > 100) {
+      throw new Error("Please set typeDepth to a reasonable value; the default is 9.");
```

Comment:
This line also needs a test

```suggestion
      throw new Error('Please set typeDepth to a reasonable value; the default is 9.');
```